src/graphql/types/Query/organization.ts

```
@@ -64,3 +83,64 @@
+						issues: error.issues.map((issue: any) => ({
+							argumentPath: issue.path,
+							message: issue.message,
+						})),
+					},
+				});
+			}
+
+			const customFields =
+				await ctx.drizzleClient.query.customFieldsTable.findMany({
+					where: (fields, operators) =>
+						parsedArgs && parsedArgs.input.organizationId
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)